### ISSUE TITLE
Any way to work caching into the build step?

### ISSUE THREAD

**Issue by warmwaffles** [dscout/wkhtmltopdf-buildpack#13]
I would love to see my builds go faster. 

https://devcenter.heroku.com/articles/buildpack-api#caching

I would do a PR but I'm not sure how I would go about testing it personally. I assume I can fork and point the build pack at my repo.

**Comment by sorentwo** [comment]
You can definitely fork and point the buildpack to your new repository. Just update the link in `.buildpacks` and it'll be fine. I believe you can also reference an exact SHA as well, similar to:

```
https://github.com/heroku/heroku-buildpack-nodejs.git#0198c71daa8
https://github.com/heroku/heroku-buildpack-ruby.git#v86
```

We're actually using this again now though, so I'm interested in getting the `CACHE_DIR` working myself.

**Comment by stevenharman** [comment]
I implemented caching in #15, which was merged earlier today. I'd recommend we close this Issue.

**Comment by sorentwo** [comment]
Right you are!
